drivers/vdpa/vfe/rte_vhost.h

```diff
@@ -0,0 +1,1073 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2010-2017 Intel Corporation
+ */
+
+#ifndef _RTE_VHOST_H_
+#define _RTE_VHOST_H_
```

Comment:
this file should not be added. use the existing vhost header file.

Reply:
if we use existing vhost header file, there will be a lot of redefine.
because virtio_pci.h also define some virtio macro, and vhost header include linux header file which also define virtio macro.

In file included from ../drivers/vdpa/vfe/virtio_pci.h:15,
                 from ../drivers/vdpa/vfe/vfe_vdpa.c:23:
../drivers/vdpa/vfe/virtio.h:48: warning: "VIRTIO_F_IOMMU_PLATFORM" redefined
   48 | #define VIRTIO_F_IOMMU_PLATFORM 33
      | 
In file included from /usr/include/linux/vhost_types.h:16,
                 from /usr/include/linux/vhost.h:14,
                 from ../drivers/vdpa/vfe/rte_vhost.h:26,
                 from ../drivers/vdpa/vfe/vfe_vdpa.c:17: